build.gradle

```diff
@@ -40,10 +43,17 @@ allprojects {
         mavenCentral()
         maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
     }
+
+    tasks.withType(JavaCompile) {
+        options.encoding = 'UTF-8'
+    }
+
+    tasks.withType(Test) {
+        systemProperty 'file.encoding', 'UTF-8'
+    }
```

Comment:
Ah, I just saw the UTF-8 getting set up here. This is a good place to set this. I'll resolve my earlier comments.

Reply:
The idea plugin is still used, it's now in a better spot next to eclipse's apply.


https://github.com/libgdx/libgdx/blob/3defef117ac6ffd328788d634fc2162c9f300525/build.gradle#L20

Reply:
Great! That does seem like a better place.